doc/classes/AnimationNode.xml

```diff
@@ -27,7 +27,7 @@
 		<method name="_get_child_nodes" qualifiers="virtual const">
 			<return type="Dictionary" />
 			<description>
-				When inheriting from [AnimationRootNode], implement this virtual method to return all children animation nodes in order as a [code]name: node[/code] dictionary.
+				When inheriting from [AnimationRootNode], implement this virtual method to return all child animation nodes in order as a [code]name: node[/code] dictionary.
```

Comment:
```suggestion
				When inheriting from [AnimationRootNode], implement this virtual method to return all descendant animation nodes in order as a [code]name: node[/code] dictionary.
```

Instead of children descendant might be a better word?

Reply:
I haven't checked which behaviour it did.